ibc-testkit/tests/core/ics04_channel/timeout.rs

```diff
@@ -45,7 +47,11 @@ fn fixture() -> Fixture {
 
     let msg_proof_height = 2;
     let msg_timeout_height = 5;
-    let timeout_timestamp = Timestamp::now().nanoseconds();
+    let timeout_timestamp = ctx
+        .latest_timestamp()
+        .sub(core::time::Duration::from_secs(3))
+        .expect("no overflow")
+        .nanoseconds();
```

Comment:
Is this the way around the fact that client and context times are not synced? Should this be reverted once we finalize refactoring?

Reply:
Here, the timeout timestamp needs to be older than the client timestamp. Currently, the client takes its host's latest timestamp as the default timestamp at initialization. That's why I use `ctx.latest_timestamp()`.

I added 1de553c to imp this logic.

Reply:
Actually, the checks for the Tendermint light client are strongly coupled with timestamps. so we should be careful of the order of the mock type initialization - as it affects the timestamp order (in the default case).